app/services/editions/publishing/EditionsBucket.scala

```diff
@@ -20,20 +20,24 @@ object EditionsBucket {
     metadata
   }
 
-  def createPutObjectRequest(bucketName: String, issue: PublishableIssue): PutObjectRequest = {
-    val key = EditionsBucket.createKey(issue)
+  def createPutObjectRequest[T:Writes](bucketName: String, key: String, issue: T): PutObjectRequest = {
     val issueJson = Json.stringify(Json.toJson(issue))
     new PutObjectRequest(bucketName, key, new StringInputStream(issueJson), EditionsBucket.objectMetadata)
   }
 }
 
-class EditionsBucket(s3Client: AmazonS3, bucketName: String) {
-  def putIssue(issue: PublishableIssue): PutObjectResult = {
-    val request = EditionsBucket.createPutObjectRequest(bucketName, issue)
+class EditionsBucket(s3Client: AmazonS3, bucketName: String) extends PublicationTarget {
+  def putIssue(issue: PublishableIssue): Unit = {
+    val key = EditionsBucket.createKey(issue)
+    putIssueJson(issue, key)
+  }
+
+  override def putIssueJson[T: Writes](content: T, key:String): Unit = {
+    val request = EditionsBucket.createPutObjectRequest(bucketName, key, content)
     s3Client.putObject(request)
   }
 
-  def putEditionsList(rawJson: String) = {
+  def putEditionsList(rawJson: String): Unit = {
```

Comment:
Hmmmmm thinking that this could be redundant, can we use `putIssueJson` instead?